src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,8 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
you already have classnames in package.json use it instead of conditions
https://github.com/JedWatson/classnames